diff -urN tf-40s1-org/tf-lib/complete.tf tf-40s1/tf-lib/complete.tf
--- tf-40s1-org/tf-lib/complete.tf	Sat Mar  6 23:43:27 1999
+++ tf-40s1/tf-lib/complete.tf	Wed Jul 31 11:40:34 2002
@@ -7,7 +7,7 @@
 ;; To use:  /load this file, and optionally store a list of words in
 ;; %{completion_list}.  For example, add this to your ~/.tfrc file:
 ;;
-;;    /load complete.tf
+;;    /load completion.tf
 ;;    /set completion_list=Hawkeye TinyFugue ftp.tcp.com
 ;;
 ;; Completion keys:
@@ -35,7 +35,7 @@
 
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 
-/loaded __TFLIB__/complete.tf
+/loaded __TFLIB__/completion.tf
 
 /require lisp.tf
 
diff -urN tf-40s1-org/tf-lib/filexfer.tf tf-40s1/tf-lib/filexfer.tf
--- tf-40s1-org/tf-lib/filexfer.tf	Sat Mar  6 23:43:27 1999
+++ tf-40s1/tf-lib/filexfer.tf	Wed Jul 31 11:42:00 2002
@@ -1,7 +1,7 @@
 ;;;; File transfer macros
 ;; see "/help /putfile" and "/help /getfile".
 
-/loaded __TFLIB__/filexfer.tf
+/loaded __TFLIB__/file-xfer.tf
 
 /def -i putfile = /putfile_MUCK %*
 
diff -urN tf-40s1-org/tf-lib/finger.tf tf-40s1/tf-lib/finger.tf
--- tf-40s1-org/tf-lib/finger.tf	Sat Mar  6 23:43:27 1999
+++ tf-40s1/tf-lib/finger.tf	Wed Jul 31 11:45:43 2002
@@ -6,7 +6,7 @@
 
 ; This is more complicated than it needs to be, just to make it act nicely.
 
-; The empty WORLD hooks override normal hooks (like those in world-q.tf)
+; The empty WORLD hooks override normal hooks (like those in worldqueue.tf)
 ; that we don't want during finger.
 
 /def -i finger = \
diff -urN tf-40s1-org/tf-lib/kb-bash.tf tf-40s1/tf-lib/kb-bash.tf
--- tf-40s1-org/tf-lib/kb-bash.tf	Sat Mar  6 23:43:27 1999
+++ tf-40s1/tf-lib/kb-bash.tf	Wed Jul 31 11:40:55 2002
@@ -1,9 +1,9 @@
 ;;;; Bash-like keyboard bindings
 
-/loaded __TFLIB__/kb-bash.tf
+/loaded __TFLIB__/bind-bash.tf
 
 /require kbfunc.tf
-/require complete.tf
+/require completion.tf
 
 ;; The commented-out lines are either unimplemented, already defined,
 ;; or conflict with pre-existing bindings.
diff -urN tf-40s1-org/tf-lib/kb-emacs.tf tf-40s1/tf-lib/kb-emacs.tf
--- tf-40s1-org/tf-lib/kb-emacs.tf	Sat Mar  6 23:43:27 1999
+++ tf-40s1/tf-lib/kb-emacs.tf	Wed Jul 31 11:39:22 2002
@@ -1,6 +1,6 @@
 ;;;; emacs style keybindings for TinyFugue
 
-/loaded __TFLIB__/kb-emacs.tf
+/loaded __TFLIB__/bind-emacs.tf
 
 /def -i -b"^a"		= /dokey_home
 /def -i -b"^b"		= /dokey_left
diff -urN tf-40s1-org/tf-lib/psh.tf tf-40s1/tf-lib/psh.tf
--- tf-40s1-org/tf-lib/psh.tf	Sat Mar  6 23:43:27 1999
+++ tf-40s1/tf-lib/psh.tf	Wed Jul 31 11:43:11 2002
@@ -4,7 +4,7 @@
 ;; bug:  barfs on suspend (^Z) because SHELL is really a child of /bin/sh,
 ;; which doesn't have job control.  Workaround:  don't ^Z during /psh.
 
-/loaded __TFLIB__/psh.tf
+/loaded __TFLIB__/pref-shell.tf
 
 /def -i psh = \
     /if ( ({#} > 0) & (SHELL !~ "") & (SHELL !~ "/bin/sh") ) \
diff -urN tf-40s1-org/tf-lib/spc-page.tf tf-40s1/tf-lib/spc-page.tf
--- tf-40s1-org/tf-lib/spc-page.tf	Sat Mar  6 23:43:27 1999
+++ tf-40s1/tf-lib/spc-page.tf	Wed Jul 31 11:44:09 2002
@@ -4,7 +4,7 @@
 ;; I personally don't like it, but you might if you can't get the hang of
 ;; pressing TAB.  To enable space-page, just load this file.
 
-/loaded __TFLIB__/spc-page.tf
+/loaded __TFLIB__/space_page.tf
 
 /def -i pager = \
     /purge -ib" "%; \
diff -urN tf-40s1-org/tf-lib/spedwalk.tf tf-40s1/tf-lib/spedwalk.tf
--- tf-40s1-org/tf-lib/spedwalk.tf	Sat Mar  6 23:43:27 1999
+++ tf-40s1/tf-lib/spedwalk.tf	Wed Jul 31 11:36:38 2002
@@ -9,7 +9,7 @@
 ;;;; enabled, typing "ne3ses" will send "n", "e", "s", "s", "s", "e", "s".
 
 
-/loaded __TFLIB__/spedwalk.tf
+/loaded __TFLIB__/speedwalk.tf
 
 /eval \
     /def -i speedwalk = \
diff -urN tf-40s1-org/tf-lib/stack-q.tf tf-40s1/tf-lib/stack-q.tf
--- tf-40s1-org/tf-lib/stack-q.tf	Sat Mar  6 23:43:27 1999
+++ tf-40s1/tf-lib/stack-q.tf	Wed Jul 31 11:45:03 2002
@@ -2,7 +2,7 @@
 ;;;; Lets you keep stacks and queues of words.  /pop and /dequeue give their
 ;;;; results via /echo; use $(/pop) and $(/dequeue) to capture their results.
 
-/loaded __TFLIB__/stack-q.tf
+/loaded __TFLIB__/stack-queue.tf
 
 /require lisp.tf
 
diff -urN tf-40s1-org/tf-lib/tf-help tf-40s1/tf-lib/tf-help
--- tf-40s1-org/tf-lib/tf-help	Sat Mar  6 23:43:27 1999
+++ tf-40s1/tf-lib/tf-help	Wed Jul 31 11:46:04 2002
@@ -1147,7 +1147,7 @@
 
   Usage: 
 
-  [1m/REQUIRE[22;0m filexfer.tf
+  [1m/REQUIRE[22;0m file-xfer.tf
 
   [1m/GETFILE_MUCK[22;0m <[4mfile[24m> [<[4mremote-file[24m>]
   [1m/GETFILE_LP[22;0m <[4mfile[24m> [<[4mremote-file[24m>]
@@ -2333,7 +2333,7 @@
 
   Usage: 
 
-  [1m/REQUIRE[22;0m filexfer.tf
+  [1m/REQUIRE[22;0m file-xfer.tf
 
   [1m/PUTFILE_MUCK[22;0m <[4mfile[24m> [<[4mremote-file[24m>]
   [1m/PUTFILE_LP[22;0m <[4mfile[24m> [<[4mremote-file[24m>]
@@ -3938,12 +3938,12 @@
 
   Usage: 
 
-  [1m/REQUIRE[22;0m complete.tf
+  [1m/REQUIRE[22;0m completion.tf
 
   [1m/complete[22;0m 
   ____________________________________________________________________________
 
-  Loading complete.tf will define the following keybindings: 
+  Loading completion.tf will define the following keybindings: 
 
   ESC TAB 
           complete from context, input [1mhistory[22;0m, or [1m%{completion_list}[22;0m.  
@@ -3969,7 +3969,7 @@
   in which you can store a list of any words you want to be able to complete.  
 
   You can also define your own types of completion.  See the 
-  [1m%{TFLIBDIR}[22;0m/complete.tf file for more information.  
+  [1m%{TFLIBDIR}[22;0m/completion.tf file for more information.  
 
   See: [1m/bind[22;0m, [1mtfrc[22;0m 
 
@@ -5356,8 +5356,8 @@
 
 Other key bindings
 #kbstack.tf
-#kb-bash.tf
-#kb-emacs.tf
+#bind-bash.tf
+#bind-emacs.tf
 #kbregion.tf
 #cut
 #paste
@@ -5368,9 +5368,9 @@
   Some additional keyboard operations can be defined by [1m/load[22;0ming these library 
   files: 
 
-  kb-bash.tf 
+  bind-bash.tf 
           keybindings like those in bash 
-  kb-emacs.tf 
+  bind-emacs.tf 
           keybindings like those in emacs 
   kbregion.tf 
           cut-and-paste operations 
@@ -6406,7 +6406,7 @@
 
   You can switch between [1mforeground[22;0m [1msockets[22;0m with the [1m/fg[22;0m command; the next 
   [1msocket[22;0m and previous [1msocket[22;0m keys, which default to ESC-b and ESC-f, and can 
-  be modified with [1m/bind[22;0m; and, when the file world-q.tf is loaded, with the 
+  be modified with [1m/bind[22;0m; and, when the file worldqueue.tf is loaded, with the 
   ESC-w keybinding.  
 
   If the [1m%{quitdone}[22;0m flag is on, and you disconnect from all worlds (either 
@@ -7905,13 +7905,13 @@
   alias.tf 
           [1m/alias[22;0m, etc: create commands without '/'.  
   at.tf   [1m/at[22;0m: execute commands at a specified time.  
-  complete.tf 
+  completion.tf 
           Automagic word completion.  
-  filexfer.tf 
+  file-xfer.tf 
           [1m/putfile[22;0m, [1m/getfile[22;0m: transfer files to/from a mud.  
-  kb-bash.tf 
+  bind-bash.tf 
           Keybindings similar to those in bash.  
-  kb-emacs.tf 
+  bind-emacs.tf 
           Keybindings similar to those in emacs.  
   kb-os2.tf 
           Extra default key bindings for OS/2 keyboards.  
@@ -7920,14 +7920,14 @@
   kbfunc.tf 
           Macros used by kbbind.tf.  
   map.tf  Mapping commands (like tintin).  
-  psh.tf  [1m/psh[22;0m: like [1m/sh[22;0m, but uses your favorite shell.  
+  pref-shell.tf  [1m/psh[22;0m: like [1m/sh[22;0m, but uses your favorite shell.  
   [1mquoter.tf[22;0m 
           Various quoting [1mmacros[22;0m.  
   rwho.tf 
           Remote WHO from a mudwho server.  
   spc-page.tf 
           Old-style SPACE key scrolling at [1m--More--[22;0m prompt.  
-  spedwalk.tf 
+  speedwalk.tf 
           Single character movement (like tintin).  
   spell.tf 
           Spelling checker.  
@@ -7938,7 +7938,7 @@
   tr.tf   [1m/tr[22;0m: character translation 
   watch.tf 
           [1m/watch[22;0m: Watch for a particular player.  
-  world-q.tf 
+  worldqueue.tf 
           Keep track of active worlds.  
 
   There are also other files, not listed here.  
diff -urN tf-40s1-org/tf-lib/tf-help.idx tf-40s1/tf-lib/tf-help.idx
--- tf-40s1-org/tf-lib/tf-help.idx	Sat Mar  6 23:43:27 1999
+++ tf-40s1/tf-lib/tf-help.idx	Wed Jul 31 11:38:58 2002
@@ -595,8 +595,8 @@
 220365&kbbind.tf
 220376&keybindings
 228022#kbstack.tf
-228034#kb-bash.tf
-228046#kb-emacs.tf
+228034#bind-bash.tf
+228046#bind-emacs.tf
 228059#kbregion.tf
 228072#cut
 228077#paste
diff -urN tf-40s1-org/tf-lib/tintin.tf tf-40s1/tf-lib/tintin.tf
--- tf-40s1-org/tf-lib/tintin.tf	Sat Mar  6 23:43:27 1999
+++ tf-40s1/tf-lib/tintin.tf	Wed Jul 31 11:37:03 2002
@@ -10,7 +10,7 @@
 ;;; Some useful stuff is stored in other files.
 /require tick.tf
 /require alias.tf
-/require spedwalk.tf
+/require speedwalk.tf
 /require map.tf
 
 /def -i action	= /trig %*
@@ -69,7 +69,7 @@
 				/echo $${world_name}: %%*%;\
 		/endif
 
-;speedwalk	(see spedwalk.tf)
+;speedwalk	(see speedwalk.tf)
 /def -i split	= /isize %{1-3}%; /visual on
 ;subs		(see /substitute)
 ;substitute	(see /substitute)
diff -urN tf-40s1-org/tf-lib/world-q.tf tf-40s1/tf-lib/world-q.tf
--- tf-40s1-org/tf-lib/world-q.tf	Sat Mar  6 23:43:27 1999
+++ tf-40s1/tf-lib/world-q.tf	Wed Jul 31 11:46:16 2002
@@ -10,9 +10,9 @@
 ;;; empty, ``ESC w'' will switch you to the last world you visited that is
 ;;; still connected.
 
-/loaded __TFLIB__/world-q.tf
+/loaded __TFLIB__/worldqueue.tf
 
-/require stack-q.tf
+/require stack-queue.tf
 /require lisp.tf
 
 /if (active_worlds =~ "") /set active_worlds=%; /endif
diff -urN tf-40s1-org/unix/unix.mak tf-40s1/unix/unix.mak
--- tf-40s1-org/unix/unix.mak	Sat Mar  6 23:43:28 1999
+++ tf-40s1/unix/unix.mak	Wed Jul 31 11:49:17 2002
@@ -98,20 +98,6 @@
 	if [ -n "$$old" ]; then \
 	    echo "## WARNING: Obsolete files found in $(LIBDIR): $$old"; \
 	fi
-	@echo '## Creating links so old library names still work...'
-#	@# note: ln -sf isn't portable.
-	@cd $(LIBDIR); \
-	rm -f bind-bash.tf;    ln -s  kb-bash.tf   bind-bash.tf;    \
-	rm -f bind-emacs.tf;   ln -s  kb-emacs.tf  bind-emacs.tf;   \
-	rm -f completion.tf;   ln -s  complete.tf  completion.tf;   \
-	rm -f factorial.tf;    ln -s  factoral.tf  factorial.tf;    \
-	rm -f file-xfer.tf;    ln -s  filexfer.tf  file-xfer.tf;    \
-	rm -f local.tf.sample; ln -s  local-eg.tf  local.tf.sample; \
-	rm -f pref-shell.tf;   ln -s  psh.tf       pref-shell.tf;   \
-	rm -f space_page.tf;   ln -s  spc-page.tf  space_page.tf;   \
-	rm -f speedwalk.tf;    ln -s  spedwalk.tf  speedwalk.tf;    \
-	rm -f stack_queue.tf;  ln -s  stack-q.tf   stack_queue.tf;  \
-	rm -f worldqueue.tf;   ln -s  world-q.tf   worldqueue.tf;
 
 makehelp: makehelp.c
 	$(CC) $(CFLAGS) -o makehelp makehelp.c
